src/NuGet.Clients/NuGet.PackageManagement.UI/NuGet.PackageManagement.UI.csproj

```diff
@@ -35,6 +35,7 @@
     </Reference>
     <Reference Include="PresentationCore" />
     <Reference Include="PresentationFramework" />
+    <Reference Include="PresentationFramework.Aero" />
```

Comment:
why do you need this? from what i remember, this contains themes for win7/xp

Reply:
The `ListBoxChrome` is contained in this dll, this is the only way I found it was possible to update the background of the `ListView` when disabled.